spark/src/test/scala/com/alibaba/graphar/TestReader.scala

```diff
@@ -79,35 +82,48 @@ class ReaderSuite extends AnyFunSuite {
 
     // test reading a single property chunk
     val single_chunk_df = reader.readVertexPropertyChunk(property_group, 0)
-    assert(single_chunk_df.columns.size == 3)
+    assert(single_chunk_df.columns.length == 3)
     assert(single_chunk_df.count() == 100)
-
-    // test reading chunks for a property group
-    val property_df = reader.readVertexPropertyGroup(property_group, false)
-    assert(property_df.columns.size == 3)
+    val cond = "gender = 'female'"
+    var df_pd = single_chunk_df.select("firstName","gender").filter(cond)
+    df_pd.explain()
```

Comment:
Could you please include the resulting physical plan in the comments? This would effectively demonstrate the filter pushdown in a more intuitive manner.

Reply:
Ok, I will apply plan on it.